### PR TITLE
[Needs live testing] Removes sleep from process() in bullet code

### PR DESCRIPTION
**Pull request by MLGTASTICa**
## About The Pull Request
Removes sleeping in process code for bullets , unlikely to fix the issues people are having but its always worth a shot.

## Why It's Good For The Game
If this fixes the lag it'd be preety nice.

## Changelog
:cl:
fix: Changed sleep() to spawn() in bullet process code .
/:cl:

### CODE REVIEW

**Comment by mattroks101** [comment]
I don't think that's going to fix anything.

File: code/modules/projectiles/projectile.dm [hunk 1]
```diff
@@ -429,7 +429,9 @@
 			tracer_effect(effect_transform)
 
 		if(!hitscan)
-			sleep(step_delay)	//add delay between movement iterations if it's not a hitscan weapon
+			spawn(step_delay) goto LOOP_START 	//add delay between movement iterations if it's not a hitscan weapon
+		else
+			goto LOOP_START
```
Comment (drexample) [inline code/modules/projectiles/projectile.dm:434]: Goto moment

**Review by drexample — CHANGES_REQUESTED** [review]

**Comment by MLGTASTICa** [comment]
> I don't think that's going to fix anything.

Not sure myself either , but it might help , since sleep makes the bullet instance sleep . while spawn and goto keep it running , might be less resource intensive than pausing , unpausing , pausing , unpausing every process

**Comment by drexample** [comment]
Absolutely not

**Pull request closed without merging**